content/tutorials/proptypes.md

```diff
@@ -1,27 +1,29 @@
 ---
-title: 'Proptypes'
+title: 'Working With Element Proptypes Tutorial'
+metaTitle: 'Working With Element Proptypes Tutorial'
+metaDescription: 'In this 15 minute tutorial, you will learn how to create an Element block with configurable light and dark modes.'
 ---
 
-In this 15 minute tutorial you will learn how to create a block with a configurable light and dark modes.
+In this 15 minute tutorial, you will learn how to create a block with configurable light and dark modes.
 
 ![Preview](result-preview.png)
 
 ## Prerequisites
 
-- [Basic understanding of blocks](explanations/element-concepts)
-- [Element CLI installed](/how-to/env-setup)
+- [Element Concepts](/explanations/element-concepts)
+- [Element Setup](/how-to/env-setup)
```

Comment:
```suggestion
- [Element CLI Setup](/how-to/env-setup)
```

Reply:
I'll change the title of that how-to page as well. Would "Environment Setup" or "Element CLI Setup" be more appropriate?

Reply:
You know what, scratch that. After reviewing the file again, I think what you have is fine.